showcase/src/app/demo/button/button.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit, Inject, ViewEncapsulation } from "@angular/core";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+
+@Component({
+	selector: "showcase-demo-button",
```

Comment:
I'm not sure these lines are correctly formatted (indentation) ... could you please run the "prettier-check" command to auto format all files?

Reply:
Yes I already did it, so I guess the indentation is correct. Seems ok in my editor...

Reply:
Ok